tests/robottelo/test_decorators.py

```diff
@@ -209,39 +183,34 @@ def bomb(_):
 
 class RunOnlyOnTestCase(TestCase):
     """Tests for :func:`robottelo.decorators.run_only_on`."""
-    def setUp(self):  # noqa
-        """Backup object."""
-        self.project_backup = conf.properties.get('main.project')
-
-    def tearDown(self):  # noqa
-        """Restore backed-up object."""
-        conf.properties['main.project'] = self.project_backup
-
-    def test_project_mode_different_cases(self):  # pylint:disable=C0103
+    @mock.patch('robottelo.decorators.settings')
+    def test_project_mode_different_cases(self, settings):
         """Assert ``True`` for different cases of accepted input values
            for project / robottelo modes."""
         accepted_values = ('SAT', 'SAt', 'SaT', 'Sat', 'sat', 'sAt',
```

Comment:
These test data are intentionally added to test possible values.  This is kind of simulated ddt test.